code/socialDistribution/templates/tagtemplates/post.html

```diff
@@ -8,18 +8,18 @@
             {{ post.when }}
         </small>
     </div>
-    <div class="card-body">
+    <div class="card-body" style="justify-content: center; text-align: center;">
         <h5 class="card-title">{{ post.title }}</h5>
         <p class="card-text text-center">
             {{ post.description }}
             {{ post.content_type }}</br>
             {{ post.content_text }}</br>
             {{ post.visibility }}</br>
             {{ post.unlisted }}</br>
+            {% if post.content_media %}
+                <img src="{{ post.content_media.url }}" alt="Post Media" width="70%" height="70%" style="object-fit: contain; overflow:auto;">
```

Comment:
Please move in-line styles to a CSS file. 
Image looks better, thanks!
Also check: https://getbootstrap.com/docs/5.1/content/images/#responsive-images